### PR TITLE
User profile Front end

### PR DESCRIPTION
**Pull request by Kartihan-sri**
Snackbar is implemented in the profile page and signup page (reusable snackbar component has been started on).

User profile page still has borders for each section for templating, this can be removed when the other profile views are completed.

### CODE REVIEW

**Review by ykurata — COMMENTED** [review]
In the profile edit page, I can click the gender field and select Male or Female. But the selected value isn't displayed in the gender field. Maybe need a small change??

**Comment by Kartihan-sri** [comment]
Done. Good find!

File: client/src/App.js [hunk 1]
```diff
@@ -17,7 +18,8 @@ function App() {
         <Route exact path="/" component={LandingPage} />
         <Route path="/login" component={LoginPage} />
         <Route path="/signup" component={SignUpPage} />
-        <Route path="/profile" component={ProfilePage} />
+        <Route path="/profile" component={EditProfilePage} />
+        <Route path="/profile-payment" component={ProfilePayment} />
```
Comment (CometS1) [inline client/src/App.js:23]: You should change this route path and associated routes to just "payment", to keep a consistent routing style

File: client/src/pages/EditProfile.js [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+import Button from "@material-ui/core/Button";
+
+import SimpleSnackbar from "./snackbar";
+import { Snackbar, IconButton } from "@material-ui/core";
+
+const initalState = {
+  firstName: "",
+  lastName: "",
+  gender: "",
+  dob: "",
```
Comment (CometS1) [inline client/src/pages/EditProfile.js]: Can you change dob to birthDate? It would make it easier to know what the variable is

**Review by ykurata — APPROVED** [review]
Looks good!

**Review by CometS1 — APPROVED** [review]
Seems good for now. You may want to look into either making each Grid item in the pages smaller and easier to read, either by using reusable code where you can place the title and values of each individual element, or just by reducing the code.

**Pull request merged**